lib/active_shipping/carriers/fedex.rb

```diff
@@ -153,8 +161,131 @@ def find_tracking_info(tracking_number, options = {})
       parse_tracking_response(xml, options)
     end
 
+
+    # Get Shipping labels
+    # Caveats:
+    #  - Only supports singlular packages
+    #
+    def create_shipment(origin, destination, packages, options = {})
+      options = @options.update(options)
+      packages = Array(packages)
+
+      begin
+        request = build_shipment_request(origin, destination, packages, options)
+        logger.debug(request) if logger
+
+        logger.debug(confirm_response) if logger
+
+        response = commit(save_request(request), (options[:test] || false))
+        parse_ship_response(response)
+      rescue RuntimeError => e
+        raise "Could not obtain shipping label. #{e.message}."
```

Comment:
Under what circumstances would this happen in practice? If this should only happen in the case of a bug in `active_shipping` why bother wrapping the message? Why `RuntimeError`, was there a specific case you were thinking of with this?

With this you lose the class information of the actual error, and the resulting raised exception is not even an activeshipping error class.